pkg/ipamd/ipamd.go

```diff
@@ -109,6 +109,10 @@ const (
 	// eniNoManageTagKey is the tag that may be set on an ENI to indicate ipamd
 	// should not manage it in any form.
 	eniNoManageTagKey = "node.k8s.amazonaws.com/no_manage"
+
+	// whitelistedAccountIds is the account ids which are allowed to track ENI.
+	// account ids should be delimited by comma
+	envWhitelistedAccountIds = "ALLOWED_ENI_TRACKING_IDS"
```

Comment:
The comment was a bit hard to read. How about:
```go
	// envWhitelistedAccountIds is the account IDs which are allowed to call EC2.
	// Account IDs should be a comma separated list.
	envWhitelistedAccountIds = "ALLOWED_ENI_TRACKING_IDS"
```

Reply:
Will update. Thanks.